projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DeployJob.tsx

```diff
@@ -110,6 +112,7 @@ export async function showCreateDeploymentDialog(statusButton: StatusButton) {
             buttons: [Dialog.okButton()]
           });
         }
+        docManager?.open('vdk_logs.txt');
```

Comment:
Export as constant somewhere in common place please. 

Also should we not rename it to vdk_run.log ? We can do that in separate change? I am just asking if you have changed your mind about that ?

Reply:
@antoniivanov I will do the name change in separate PR since it will require some changes in the getting-started notebook too